### DISCUSSION TITLE
'UNSUPPORTED' message

### DISCUSSION THREAD

**Discussion by JayBeest** [codam-coding-college/MLX42#29] [General]
When running MLX42 on M1 I get the following message:

UNSUPPORTED (log once): POSSIBLE ISSUE: unit 1 GLD_TEXTURE_INDEX_2D is unloadable and bound to sampler type (Float) - using zero texture because texture unloadable

Does anybody know where this message comes from ?
![Screenshot 2022-05-02 at 22 56 30](https://user-images.githubusercontent.com/67948678/166330014-2669a712-3398-406d-a496-ba5a777bba4d.png)

Apart from the message it runs like a dream :)

**Comment by W2Wizard** [comment]
Yeah I have seen this before recently with someone trying to running it on M1. It's been mentioned on several posts online specifically to M1 about this issue as well.

Personally, I can't reproduce it (as I don't have a fancy M1 😛). For now you can safely ignore it I suppose. It should still work just fine.

**Reply by W2Wizard** [reply]
**Update**: I now have a Macbook with an apple silicon chip, meaning i can start looking into what exactly causes this.

So far tho, it doesn't end up in non-functioning projects.

**Comment by OSS-42** [comment]
I have it on M2 too.
But window is loading just fine. Seems link to mlx_image_to_window function, as when I put it in comments, the warning message disappear.
Does it prevent any rendering from happening ?
